<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>火灾监测系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="previewGroup">
      <property name="title">
       <string>图像预览</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string>无图像数据</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="controlGroup">
      <property name="title">
       <string>处理识别图像</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="processButton">
           <property name="text">
            <string>处理图像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshButton">
           <property name="text">
            <string>刷新结果</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="simulateButton">
           <property name="text">
            <string>火灾蔓延模拟</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="searchLayout">
         <item>
          <widget class="QLineEdit" name="searchLineEdit">
           <property name="placeholderText">
            <string>输入图片ID搜索</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="resultsTable">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>预测结果</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>置信度</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>处理时间</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="prevButton">
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="analysisImageGroup">
      <property name="title">
       <string>火灾分析图像</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="riskMapLabel">
         <property name="text">
          <string>无风险地图数据</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="reportAndLogGroup">
      <property name="title">
       <string>分析报告与日志</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTextEdit" name="reportTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>无分析报告数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="logTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>无日志数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
